pkg/cvo/sync_worker.go

```diff
@@ -731,7 +733,7 @@ func (r *consistentReporter) Error(err error) {
 	copied := r.status
 	copied.Step = "ApplyResources"
 	copied.Fraction = float32(r.done) / float32(r.total)
-	if !isCancelledError(err) {
+	if !isContextError(err) {
```

Comment:
Replace this with errors.Is ?

Reply:
Which `errors.Is` we are talking about here. Is it from https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go ?

Reply:
> Which errors.Is we are talking about here. 

https://blog.golang.org/go1.13-errors